web-console/src/lib/components/pipelines/table/AvailableActions.svelte

```diff
@@ -38,8 +39,7 @@
         ]
       )
       .with('Shutdown', { SqlWarning: P.any }, () => ['start', 'delete'])
```

Comment:
Not this PR, but when it is shutdown, you can still do `/shutdown`. This is handy when I click "Check all"-box and then just want to press Shutdown.

Reply:
One exception is if there are only Shutdown pipelines selected, in which case the shutdown button shouldn't appear. (not this PR)